src/components/Header/Menu/MenuContainer.js

```diff
@@ -0,0 +1,61 @@
+import React, {useState} from 'react';
+import PropTypes from 'prop-types';
+import {Link} from 'gatsby';
+import {Menu, MenuItem, IconButton} from '@material-ui/core';
+import MenuIcon from '@material-ui/icons/Menu';
+import {useTheme} from '@material-ui/core/styles';
+
+const MenuContainer = ({menu}) => {
+  const theme = useTheme();
+  const [anchorEl, setAnchorEl] = useState(null);
+  const handleClick = (event) => setAnchorEl(event.currentTarget);
+  const handleClose = () => setAnchorEl(null);
+  return (
+    <>
+      <IconButton
+        aria-controls="simple-menu"
+        aria-haspopup="true"
+        aria-label="Dropdown Menu"
```

Comment:
Can this label be named something more appropriately? `Navigation`? Or do you think this is ok?